### ISSUE TITLE
[Feature] Privacy Manifest: UserDefaults

### ISSUE THREAD

**Issue by leofolive** [Adyen/adyen-ios#1555]
Apple is recommending declaring the use of the UserDefaults API in the privacy manifest file.
https://developer.apple.com/news/?id=r1henawx

**Describe the solution you'd like**
Create a privacy manifesto declaring all APIs and reasons.

**Additional context**
```
Used symbols in binary ./XCFrameworkIntermediates/Adyen3DS2/Adyen3DS2.framework/Adyen3DS2: NSUserDefaults
Used symbols in binary ./XXXX.app/Frameworks/Adyen3DS2.framework/Adyen3DS2: NSUserDefaults
```

UserDefault API: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278401

**Comment by goergisn** [comment]
Hi @LFMAKER ,
I believe this is a duplicate of https://github.com/Adyen/adyen-ios/issues/1506

We're planning to do a release within the next days.

Which SDK version are you on?

**Comment by leofolive** [comment]
@goergisn Thanks! I'm using version 5.4.1 through https://github.com/Adyen/adyen-react-native.
@descorp Can we release a version of the Adyen React Native SDK when we release this version of the iOS SDK?

**Comment by descorp** [comment]
Hey @LFMAKER 

Sure - we always trying to update to the latest version of native SDKs :)

**Comment by goergisn** [comment]
Hi @LFMAKER we just released `5.7.0` so closing this ticket for now.
Feel free to open a new issue on the [react native repository](https://github.com/Adyen/adyen-react-native/issues)
